$OpenBSD$

From pkgsrc:
-prefix uses of "exception" with "std::" to avoid conflict with
a similar definition in new "boost".

--- src/core/modules/ConstellationMgr.cpp.orig	Sat Mar 10 07:00:24 2012
+++ src/core/modules/ConstellationMgr.cpp	Thu May 10 19:18:06 2012
@@ -442,7 +442,7 @@ void ConstellationMgr::loadLinesAndArt(const QString &
 				{
 					texturePath = StelFileMgr::findFile(QString("textures/")+texfile);
 				}
-				catch(exception& e2)
+				catch(std::exception& e2)
 				{
 					qWarning() << "ERROR: could not find texture, " << texfile << ": " << e2.what();
 				}
